msi/GitProduct.wxs

```diff
@@ -295,9 +315,12 @@
       <Custom Action="ExecConfigureCoreAutoCrlf" Before="ExecConfigureFsCache">
         <![CDATA[REMOVE <> "ALL"]]>
       </Custom>
-      <Custom Action="ExecConfigureFsCache" Before="ExecPostInstallBat">
+      <Custom Action="ExecConfigureFsCache" Before="ExecConfigureGcm">
         <![CDATA[REMOVE <> "ALL" AND ENABLEFSCACHE = "true"]]>
       </Custom>
+      <Custom Action="ExecConfigureGcm" Before="ExecPostInstallBat">
+        <![CDATA[REMOVE <> "ALL" AND WIX_IS_NETFRAMEWORK_451_OR_LATER_INSTALLED AND ENABLEGCM]]>
```

Comment:
Also, the condition `REMOVE <> "ALL"` merely means, _&lt;do this thing> **only if** we're not completely uninstalling the product_ (or, said equivalently, _&lt;don't do this thing> if we're removing the product_)